app/controllers/users/callbacks_controller.rb

```diff
@@ -9,4 +9,15 @@ def github
 
     redirect_to profile_path, notice: 'Connected to GitHub!'
   end
+
+  def stackexchange
+    so_auth = request.env["omniauth.auth"]
+
+    current_user.update(
+      so_token: so_auth['credentials']['token'],
+      so_uid: so_auth['uid']
+    )
```

Comment:
Awesome. We can use this when we head in to the `Sinatra API` to be able to get the info we actually want and need: Badges!